python/hsfs/core/statistics_engine.py

```diff
@@ -70,10 +71,11 @@ def compute_statistics(
     @staticmethod
     def profile_statistics(metadata_instance, feature_dataframe):
         if len(feature_dataframe.head(1)) == 0:
-            raise exceptions.FeatureStoreException(
+            warnings.warn(
                 "There is no data in the entity that you are trying to compute "
                 "statistics for. A possible cause might be that you inserted only data "
-                "to the online storage of a feature group."
+                "to the online storage of a feature group.",
+                category=util.StatisticsWarning,
```

Comment:
We need to return None here, otherwise the statistics computation will go ahead.